README.md

```diff
@@ -1 +1,13 @@
-# control-energetico
\ No newline at end of file
+## Descripción del problema: Optimización del consumo energético en el hogar
+
+En el contexto actual de creciente preocupación por la sostenibilidad, el aumento de los costos energéticos y la necesidad urgente de mitigar el impacto ambiental, la optimización del consumo energético en los hogares se ha convertido en un desafío prioritario. Aunque los avances tecnológicos ofrecen nuevas oportunidades para mejorar la eficiencia energética, muchos hogares siguen sin tener una visión clara y detallada de cómo, cuándo y dónde se consume la mayor parte de la energía en sus viviendas. Esta falta de información impide la adopción de medidas efectivas que podrían reducir significativamente el consumo y, con ello, los costos asociados y la huella ecológica.
+
+## Lenguaje
+
+Para la implementación del proyecto usaré Python.

```

Comment:
Nadie ha dicho que habléis del lenguaje.

Reply:
Debes mirar todos los comentarios y resolverlos o bien cambiando (o borrando) la línea o bien pinchando en "resolver" explicando dónde lo habéis hecho.